test/sqllogictest/chbench.slt

```diff
@@ -876,14 +875,12 @@ ORDER BY ordercount DESC
 | Get %3 (l0)
 | Reduce group=()
 | | agg sum(#14)
-| Map (i64tonumeric(#0) * 0.005)
 | ArrangeBy ()
 
 %6 =
 | Join %4 %5
 | | implementation = Differential %4 %5.()
-| | demand = (#0, #1, #3)
-| Filter (i64tonumeric(#1) > #3)
+| Filter (i64tonumeric(#1) > (i64tonumeric(#2) * 0.005))
```

Comment:
This seems like a good change that will remove an entire arrangement, but I don't get how projection pushdown enabled it. Is it easy to explain?